code/go/0chain.net/core/util/columnizer.go

```diff
@@ -0,0 +1,41 @@
+package util
+
+import (
+	"fmt"
+	"reflect"
+)
+
+func Columnize[T any](objects []T) (map[string][]interface{}, error) {
+	columns := make(map[string][]interface{})
+	
+	for _, obj := range objects {
+		v := reflect.ValueOf(obj)
+
+		if v.Kind() == reflect.Ptr {
+			v = v.Elem()
+		}
+
+		if v.Kind() != reflect.Struct {
+			return nil, fmt.Errorf("columnize error: type invalid")
+		}
+
+		t := v.Type()
+
+		for fidx := 0; fidx < t.NumField(); fidx++ {
+			f := t.Field(fidx)
+			fkey := f.Tag.Get("json")
```

Comment:
I didn't disregard it totally, instead supported two more naming methods, which are gorm and snake-casing with precedence order of JSON tag then gorm then snake-casing, just in case a field needs to be named specially not following the snake-casing rules. Should this totally be disregarded ?

Reply:
I think yes, since gorm disregards it by default